internal/test/builder/builders.go

```diff
@@ -661,6 +998,65 @@ func (b *TestBootstrapTemplateBuilder) Build() *unstructured.Unstructured {
 	return b.obj
 }
 
+// BootstrapObjectBuilder holds the variables needed to build a generic BootstrapObject.
+type BootstrapObjectBuilder struct {
```

Comment:
```suggestion
type BootstrapConfigBuilder struct {
```
(+ related changes below, same for TestBootstrapObject)